test/src/tests-persistence.ts

```diff
@@ -444,21 +473,21 @@ describe('migrations and persistence-related', function() {
         done();
     });
 
-    it('should add new MPID to cookies when returned MPID does not match anything in cookies, and have empty UI and UA', function(done) {
-        mParticle._resetForTests(MPConfig);
+    it('should add new MPID to cookies when returned MPID does not match anything in cookies, and have empty UI and UA', done => {
+        window.mParticle._resetForTests(MPConfig);
```

Comment:
could this go in a beforeEach or similar?

Reply:
Possibly. We have some tests that don't need to reset the SDK, but that might require restructuring this test file. I'd rather save this for a future refactor than introduce more scope creep.